Spanner/tests/Unit/PgJsonBTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Spanner\Tests\Unit;
+
+use Google\Cloud\Spanner\PgJsonB;
+use Google\Cloud\Spanner\V1\TypeAnnotationCode;
+use Google\Cloud\Spanner\V1\TypeCode;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group spanner
+ * @group spanner-pgjsonb
+ */
+class PgJsonBTest extends TestCase
+{
+    /**
+     * @dataProvider validValueProvider
+     */
+    public function testValidValues($value, $expectedVal)
+    {
+        $obj = new PgJsonB($value);
+        $this->assertInstanceOf(PgJsonB::class, $obj);
+        $this->assertEquals($expectedVal, $obj->get());
+    }
+
+    public function validValueProvider()
+    {
+        $obj = $this->createMock('JsonSerializable');
```

Comment:
Would you be open to using `$this->prophesize` here? Only bring it up to help retain consistency.